### DISCUSSION TITLE
Subtitle offset at server instead of client

### DISCUSSION THREAD

**Discussion by theshoeshiner** [jellyfin/jellyfin-web#4759] [Ideas]
I have seen subtitle offset features implemented in some of the JF clients, but out of curiosity, is it not possible to implement this on the server? If a subtitle file is 10 seconds off, it stands to reason that the user would want it synced regardless of the client. Could the server not handle retaining this config and adjusting as necessary, requiring no changes to the clients? Or are server level changes just avoided?

I've checked out the code but before I waste any time (I'm a Java dev so there will be a slight learning curve) I wanted to check if this is impossible or unwanted.

**Comment by juxuanu** [comment]
This is closed. Does it mean it already works this way?

**Reply by theshoeshiner** [reply]
I created PR: https://github.com/jellyfin/jellyfin/pull/10196 for it, but have not had a chance to finish resolving the conflicts.

**Reply by juxuanu** [reply]
I'll keep an eye on it, thanks
